test/checks/shared/invalidrole.js

```diff
@@ -111,4 +111,19 @@ describe('invalidrole', function () {
     );
     assert.deepEqual(checkContext._data, ['foo', 'bar']);
   });
+
+  it('should return true if applied to an uppercase nonsensical role', function () {
+    var virtualNode = queryFixture(
+      '<div id="target" role="FOO">Contents</div>'
```

Comment:
Thanks for the test. Sorry for the delay (been super busy with the 4.9 release happening soon). So an invalid role that is capitalized would still return true if the role was lowercased. We'll want a test that shows that a valid role that is uppercase will now return false. Something like:

```html
<div id="target" role="BUTTON">Contents</div>
```

Reply:
It's alright! Everyone is busy with their work/life, so no rush, :) I did add a test that shows that a valid role that is uppercase will now return false.